contracts/Shark.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.4.18;
+
+contract Shark {
+    string[10] public species;
+    SharkObject[] public sharks;
+    mapping (uint => address) owners;
+    mapping (address=> uint) ownerSharkCount;
+    struct SharkObject {
+        uint speed;
+        uint strength;
+        string typeOfShark;
```

Comment:
You can add `winCount`, `level`, and `lossCount` after this line. All three should be numbers so using `uint` would still work.